public/src/components/mobile_verification/MobileVerification.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import { Metadata } from "grpc-web-client";
+import { DalalActionService } from "../../../proto_build/DalalMessage_pb_service";
+import {MobileVerificationForm} from "./MobileVerificationForm";
+
+export interface MobileVerificationProps {
+    sessionMd: Metadata
+    updatePhoneVerified: () => void
+}
+
+export class MobileVerification extends React.Component<MobileVerificationProps,any> {
+    constructor(props: MobileVerificationProps) {
+        super(props);
+     }
+     updateStateofApp = () =>{
+          this.props.updatePhoneVerified();
+     }
+     render() {
+        return (
+            <div>
+            <div id="mobile-container-large">
+                <div className="ui middle aligned center aligned grid custom-checker ">
+                    <div className="fakecolumn column">
+                        <h2 className="ui inverted header">
+                            <div className="content" >
+                                Enter your registered Mobile Number
+                            </div>
+                        </h2>
+                        <MobileVerificationForm sessionMd={this.props.sessionMd} updatePhoneVerifiedStateofApp={this.updateStateofApp} />
```

Comment:
Change `updatePhoneVerifiedStateofApp` to `updatePhoneVerifiedState` and `updateStateofApp ` to `handleIsPhoneVerified`.